benefits/eligibility/views.py

```diff
@@ -75,58 +75,49 @@ def start(request):
 
     button = viewmodels.Button.primary(text=_("eligibility.buttons.continue"), url=reverse(ROUTE_CONFIRM))
 
-    media = [
-        dict(
-            icon=viewmodels.Icon("bankcardcheck", pgettext("image alt text", "core.icons.bankcardcheck")),
-            heading=_("eligibility.pages.start.bankcard.title"),
-            details=_("eligibility.pages.start.bankcard.text"),
-            links=[
+    # define the verifier-specific required item
+    identity_item = dict(
+        icon=viewmodels.Icon("idcardcheck", pgettext("image alt text", "core.icons.idcardcheck")),
+        heading=_(verifier.start_item_name),
+        details=_(verifier.start_item_description),
+    )
+
+    if verifier.is_auth_required:
+        if verifier.uses_auth_verification:
+            identity_item["links"] = [
                 viewmodels.Button.link(
                     classes="btn-text btn-link",
-                    text=_("eligibility.pages.start.bankcard.button[0].link"),
-                    url=f"{reverse(ROUTE_HELP)}#payment-options",
+                    text=_("eligibility.pages.start.oauth.link_text"),
+                    url=f"{reverse(ROUTE_HELP)}#login-gov",
                 ),
-            ],
-        ),
-    ]
-
-    if verifier.is_auth_required:
-        media.insert(
-            0,
-            dict(
-                icon=viewmodels.Icon("idcardcheck", pgettext("image alt text", "core.icons.idcardcheck")),
-                heading=_("eligibility.pages.start.oauth.heading"),
-                details=_("eligibility.pages.start.oauth.details"),
```

Comment:
These `eligibility.pages.start.oauth.*` IDs need to go into the data migrations, right? They were deleted from the PO files.

Reply:
They should be deleted because they're duplicates of ones in the data migrations. (`eligibility.pages.start.oauth.heading` was a duplicate of `eligibility.pages.start.oauth.items[0].title` and `eligibility.pages.start.oauth.details` was a duplicate of `eligibility.pages.start.oauth.items[0].text`). 

The view was essentially ignoring the msgids configured on the `EligibilityVerifier` and using these hard-coded msgds.

Reply:
Here are entries currently in `dev`:
https://github.com/cal-itp/benefits/blob/dev/benefits/locale/en/LC_MESSAGES/django.po#L570-L580
https://github.com/cal-itp/benefits/blob/dev/benefits/locale/en/LC_MESSAGES/django.po#L157-L167

Reply:
Ah, my mistake! I misunderstood that these were dupes.

Reply:
No worries! Our old approach made it easy for clutter to build up.